src/server/server.cc

```diff
@@ -119,13 +126,23 @@ bool used_identifier(uint32_t player_id, const rules::Players& players)
     return false;
 }
 
-void Server::wait_for_players()
+void Server::wait_for_clients()
 {
     if (FLAGS_nb_clients <= 0)
         FATAL("Server started with nb_clients <= 0.");
 
+    int spectator_count = FLAGS_nb_clients - config_.player_count;
+    if (spectator_count < 0)
+        FATAL("Server must be started with at least -nb_clients %d",
```

Comment:
Suggestion: use `--foo` for flags instead of `-foo`, it looks like a minus (and it's not GNU).

Reply:
Done.